src/pages/index.js

```diff
@@ -85,3 +177,41 @@ document.getElementById("profile-edit-button").addEventListener("click", () => {
 document.querySelector(".profile__add-button").addEventListener("click", () => {
   addCardPopup.open();
 });
+
+document
+  .querySelector(".profile__pic-container")
+  .addEventListener("click", () => updateAvatarPopup.open());
+
+// Fetch user info and initial cards from the server and render them
+api
+  .getAppInfo()
+  .then(([userData, cards]) => {
+    console.log("User data:", userData);
+    console.log("Cards data:", cards);
+
+    userInfo.setUserInfo({
+      name: userData.name,
+      job: userData.about,
+      avatar: userData.avatar,
+    });
+    section.renderItems(cards);
+
+    // Upload initial cards if the server has none
+    if (cards.length === 0) {
+      initialCards.forEach((card) => {
+        api
+          .addNewCard(card)
+          .then((newCard) => {
+            console.log("New card uploaded:", newCard);
+            const cardElement = createCard(newCard);
+            section.addItem(cardElement);
+          })
+          .catch((err) => {
+            console.error("Failed to upload card:", err);
+          });
+      });
+    }
+  })
+  .catch((err) => {
+    console.error("Failed to fetch app info:", err);
```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors